rviz_default_plugins/include/rviz_default_plugins/displays/tf/tf_display.hpp

```diff
@@ -40,7 +40,7 @@
 #include <OgreQuaternion.h>
 #include <OgreVector3.h>
 
-#include "geometry_msgs/msg/transform_stamped__struct.hpp"
+#include "geometry_msgs/msg/detail/transform_stamped__struct.hpp"
```

Comment:
As an alternative this could also just include `geometry_msgs/msg/transform_stamped.hpp`?

Reply:
See 6e15711879f7d0aa1eaf7c0d6cffc0ce09397cae.